core/commands/pubsub.go

```diff
@@ -302,8 +305,13 @@ To use, the daemon must be run with '--enable-pubsub-experiment'.
 			return
 		}
 
+		var topic string
+		if len(req.Arguments()) == 1 {
```

Comment:
This will cause confusion if more topics are passed. IMO either error out or use all of them.